proxy/create_container_interceptor.go

```diff
@@ -105,7 +110,17 @@ func (i *createContainerInterceptor) setWeaveWaitEntrypoint(container *docker.Co
 		}
 	}
 
+	for _, e := range container.Entrypoint {
+		if e != weaveWaitEntrypoint[0] {
+			return nil
+		}
+	}
```

Comment:
What is this trying to achieve?  Why is it checking all entries in container.Entrypoint?